content/_data/opencollective.js

```diff
@@ -2,48 +2,106 @@
 import eleventyFetch from '@11ty/eleventy-fetch';
 import { groupBy } from 'lodash-es';
 
+const GITHUBKEY = '';
```

Comment:
@jamesnw I think we can add an env var via the Netlify UI (https://app.netlify.com/sites/oddleventy/configuration/env#environment-variables), and then check for it here with `process.env.GITHUBKEY` or whatever. Are there security concerns with that approach?

Reply:
@jgerigmeyer  I think it's fine to manage the env var that way, but am wondering about the token itself. It sounds like the ideal would be to create a Github App, and then [generate a JWT token](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app) to use as the key (which means it isn't tied to a user). Alternatively, we could generate a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) tied to a user's account.

Reply:
I'd lean toward a PAT, but would defer to @jerivas on this.

Reply:
I don't feel strongly about this, but JWT seems like the "correct" way. Ideally, we would have a GH application installed on the OddBird org to get this information. However, if that is too annoying, a PAT for our OddBot user feels adequate as well.

Reply:
Let's go with a PAT for our OddBot user. Unless someone is interested in going through the process of setting up the app for the sake of learning, I don't think it's worth the trouble. @jgerigmeyer Can you either put the OddBot login in the shared vault, or create a PAT and replace the existing GITHUB_KEY in Netlify?

Reply:
Moved login info to 1Password 👍

Reply:
Created the key with odd-bot, updated in Netlify, and added a calendar event to remind us to regenerate in a year.